app/components/candidate_interface/carry_over_banner_component.html.erb

```diff
@@ -7,7 +7,7 @@
     <p class="govuk-body">Your application did not lead to a place, but you can <%= govuk_link_to 'apply again', start_path, class: 'govuk-notification-banner__link' %></p>
   <% else %>
     <%= notification_banner.slot(:heading, text: 'Do you want to continue applying?') %>
-    <p class="govuk-body">Applications are open for courses starting academic year 2021 to 2022.</p>
+    <p class="govuk-body">Applications are open for courses starting academic year <%= RecruitmentCycle.cycle_name(RecruitmentCycle.next_year) %>.</p>
```

Comment:
Do the other hardcoded year strings above need replacing as well?

Reply:
Done a search and no more hard coded strings of this type. We fixed this due to a failing spec as a result of changing how `current_year` is calculated. Previously `current_year` could only ever be 2020 or 2021

Reply:
Are the year strings here going to be replaced in a separate PR? https://github.com/DFE-Digital/apply-for-teacher-training/pull/5067/files#diff-10974595b3796ba294e374c363e07b6e02fe0a11296857a4383ac5633f71dd41R3-L6

Reply:
@malcolmbaig Yep - The plan is to pick these up in the 'carry over' PR that's currently on-hold.